pkg/workloadendpointmanager/workloadendpoint_manager.go

```diff
@@ -324,3 +326,22 @@ func (r *workloadEndpointManager) IsIPBelongWEPCurrent(ctx context.Context, name
 
 	return isBelongWEPCurrent, nil
 }
+
+// CheckCurrentContainerID will check the WorkloadEndpoint current containerID whether is same with the given args or not
```

Comment:
CheckCurrentContainerID will check whether the the current containerID of WorkloadEndpoint is same with the given args or not